packages/analyzer/gulpfile.js

```diff
@@ -83,7 +83,7 @@ task('compile', function () {
     gulp.src('src/**/*.ts');  //.pipe(newer({dest: 'lib', ext: '.js'}));
   const tsResult =
     srcs.pipe(sourcemaps.init())
-      .pipe(typescript(tsProject, [], typescript.reporter.fullReporter()));
+      .pipe(tsProject(typescript.reporter.fullReporter()));
```

Comment:
Why this change?

Not that I'm complaining, but touching gulpfiles is usually so obnoxious that I assume someone has to have had a really good reason to have done so, so I'm curious :)

Reply:
There were some API changes going from gulp-typescript 2->4, which I needed because gulp-typescript 2 didn't support "extends".